src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(getRandomBall().toString());
+        }
     }
```

Comment:
The 'main' method creates several instances of the 'Ball' class indirectly through the 'getRandomBall' method. According to the checklist, you should create a single instance of the 'Lottery' class and use it to get random balls.